g2p_registry_rest_api/services/group.py

```diff
@@ -27,20 +29,23 @@ class GroupApiService(Component):
             )
         ],
         output_param=PydanticModel(GroupInfoOut),
-        auth="jwt",
+        auth="user",
     )
     def get(self, _id):
         """
         Get partner's information
         """
-        partner = self._get(_id)
+        grp_helper = self.env["registrant_group.rest.service.helper"]
+        partner = grp_helper._get(_id)
+        if not partner:
+            raise NotFound()
```

Comment:
The best would be to do this by overwriting the _get in your specific implementation.

Reply:
The idea for creating the `GroupApiHelper` and `IndividualApiHelper` is to be able to reuse the same code in both places. Like the processIndividual from `IndividualApiHelper` could be reused in Group creation. It separating out the processing parts of this, so that these can be individually overwritten and reused, at different places. Logic wise nothing has changed from before.

Reply:
What about using mixin then to do this?